src/main/scala/shade/memcached/internals/SpyMemcachedIntegration.scala

```diff
@@ -436,6 +468,8 @@ class SpyMemcachedIntegration(cf: ConnectionFactory, addrs: Seq[InetSocketAddres
         case CASResponse.OBSERVE_TIMEOUT =>
           CASObserveTimeout
       }
+    case x if x.isSuccess =>
```

Comment:
Is this a good final catch-all translation for success cases (if successful -> CASSuccessStatus).

I ask especially because there is already a translation to CASSuccessStatus.